projects/netgrif-components/src/lib/data-fields/text-field/html-textarea-field/html-textarea-field.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="height-100" #quillContainer>
     <quill-editor theme="snow" *ngIf="!formControlRef.disabled" [(ngModel)]="textAreaField.value"
-                  [modules]="quillModules" [bounds]="quillContainer"></quill-editor>
+                  [modules]="quillModules" [bounds]="quillContainer" placeholder="{{ 'dataField.textarea.insertText' | translate }}"></quill-editor>
```

Comment:
should this not use the placeholder attribute from the data field object?

Reply:
Done